### PR TITLE
GCI Task: [netjsonconfig] TypeError: can only concatenate list (not "str") to list

### LINKED ISSUE #118: TypeError: can only concatenate list (not "str") to list
I made a backup of two different systems (Archer C7 v2 and X86_64 Qemu guest).
Both I tried to reverse with the following lines:

```
from netjsonconfig import OpenWrt
#router = OpenWrt(native=open('./backup-g0.sim.libcom.de-2018-10-10.tar.gz'))
router = OpenWrt(native=open('./backup-OpenWrt-2018-10-10.tar.gz'))
print(router.render)
```

In both cases this results in the following error:
python reverse.py 
```
Traceback (most recent call last):
  File "reverse.py", line 3, in <module>
    router = OpenWrt(native=open('./backup-OpenWrt-2018-10-10.tar.gz'))
  File "/usr/lib/python2.7/site-packages/netjsonconfig/backends/base/backend.py", line 47, in __init__
    self.parse(native)
  File "/usr/lib/python2.7/site-packages/netjsonconfig/backends/base/backend.py", line 280, in parse
    self.to_netjson()
  File "/usr/lib/python2.7/site-packages/netjsonconfig/backends/base/backend.py", line 293, in to_netjson
    value = converter.to_netjson()
  File "/usr/lib/python2.7/site-packages/netjsonconfig/backends/base/converter.py", line 108, in to_netjson
    result = self.to_netjson_loop(block, result, index + 1)
  File "/usr/lib/python2.7/site-packages/netjsonconfig/backends/openwrt/converters/interfaces.py", line 206, in to_netjson_loop
    interface = self.__netjson_interface(block)
  File "/usr/lib/python2.7/site-packages/netjsonconfig/backends/openwrt/converters/interfaces.py", line 217, in __netjson_interface
    interface = self.__netjson_addresses(interface)
  File "/usr/lib/python2.7/site-packages/netjsonconfig/backends/openwrt/converters/interfaces.py", line 262, in __netjson_addresses
    for address in ipv4 + ipv6:
TypeError: can only concatenate list (not "str") to list
```
[new.tar.gz](https://github.com/openwisp/netjsonconfig/files/2475301/new.tar.gz)

### PR DESCRIPTION
**Pull request by stealthflame**
This Issue is in reference to the issue #118 . It would be great to have your views on the changes made.
Thank you in advance. 

Closes #118

### CODE REVIEW

File: tests/openwrt/test_interfaces.py [hunk 1]
```diff
@@ -197,6 +213,17 @@ def test_parse_missing_ipv4_mask_list(self):
         o = OpenWrt(native=native)
         self.assertEqual(o.config, self._eth0_32)
 
+    def test_parse_missing_ipv6_mask_list(self):
+        native = self._tabs("""package network
+
+config interface 'eth0'
+    option ifname 'eth0'
+    list ipaddr 'fd87::1'
```
Comment (nemesisdesign) [inline tests/openwrt/test_interfaces.py]: on OpenWRT, `ipaddr` is for ipv4, you have to use `ip6addr` to trigger your test, use the following:

```
option ip6addr '2aa1:4aaa:2aaa:1d::5/64'
```

Update the expected value accordingly as well.

**Comment by stealthflame** [comment]
Done @nemesisdesign

**Comment by nemesisdesign** [comment]
@stealthflame you don't need to close it, just push force to your branch

**Comment by stealthflame** [comment]
Done @nemesisdesign sorry for the inconvenience.

**Pull request merged**